module/behaviour/skills/HeadBehaviourSoccer/src/HeadBehaviourSoccer.hpp

```diff
@@ -49,8 +49,10 @@ namespace module::behaviour::skills {
         enum SearchState { FIXATION = 0, WAIT = 1, SEARCH = 2 };
         SearchState state = SearchState::SEARCH;
 
-        message::vision::Balls getFixationObjects(std::shared_ptr<const message::vision::Balls> vballs, bool& search);
-        message::vision::Goals getFixationObjects(std::shared_ptr<const message::vision::Goals> vgoals, bool& search);
+        message::vision::Balls getFixationObjects(const std::shared_ptr<const message::vision::Balls>& vballs,
+                                                  bool& search);
+        message::vision::Goals getFixationObjects(const std::shared_ptr<const message::vision::Goals>& vgoals,
+                                                  bool& search);
```

Comment:
Is bool an out parameter? If not why isn't it const.

Reply:
it is in fact an out parameter.